### PR TITLE
CPDEL-593: Add an option to force-send emails

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDEL-593

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

### Review Checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests
- [x] All forms have an `id` attribute on them

### How can someone see it it review app?
1. You can't.

### CODE REVIEW

File: app/models/invite_email_mentor.rb [hunk 1]
```diff
@@ -8,7 +8,9 @@ def mail_to_send
   end
 
   def send!(force_send: false)
-    can_send_automatically = user.is_on_core_induction_programme? && Time.zone.now >= INVITES_SENT_FROM
+    return unless user.is_on_core_induction_programme?
+
+    can_send_automatically = Time.zone.now >= INVITES_SENT_FROM
```
Comment (xswoodsx) [inline app/models/invite_email_mentor.rb]: Was the force send sending non-cip users an email before making this change?

Comment (kicferk1) [reply app/models/invite_email_mentor.rb]: It in principle could, yes.

Comment (kicferk1) [reply app/models/invite_email_mentor.rb]: And I got worried that I would force-send emails to a wrong user by accident :P

File: app/models/tracked_email.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ def mail_to_send
     raise NotImplementedError
   end
 
-  def send!
+  def send!(*)
```
Comment (xswoodsx) [inline app/models/tracked_email.rb:18]: This is splat operator/naked splat right? It lets you pass any number of arguments to super but we aren't calling super at any point are we? 

I'm just trying to understand why it's needed if we're overriding the method in invite_email_ect and invite_email_mentor .

**Review by xswoodsx — APPROVED** [review]
LGTM :)

**Pull request merged**